packages/datetime/src/datePicker.tsx

```diff
@@ -25,6 +25,12 @@ export interface IDatePickerProps extends IDatePickerBaseProps, IProps {
     canClearSelection?: boolean;
 
     /**
+     * Props to pass to ReactDayPicker.
+     * See API documentation [here](http://react-day-picker.js.org).
+     */
+    dayPickerBaseProps?: ReactDayPicker.Props;
```

Comment:
Let's call this `dayPickerProps`; the `Base` isn't necessary and doesn't align with existing Blueprint convention.